integration/js/app/video_test/video_test.ts

```diff
@@ -113,6 +113,90 @@ export class DemoMeetingApp implements AudioVideoObserver {
       unbindVideoElementForm.reset();
     });
 
+    const addVideoTileForm = document.getElementById('add-video-tile-form') as HTMLFormElement;
+    addVideoTileForm.addEventListener('submit', e => {
+      e.preventDefault();
+      const tileId = this.audioVideo.addVideoTile().id();
+      let videoTileIdObject = document.getElementById(`video-tile-id`);
+      videoTileIdObject.innerText = JSON.stringify(tileId, null, '  ');;
+      addVideoTileForm.reset();
+    });
+
+    const hasStartedLocalVideoTileForm = document.getElementById('has-started-local-video-tile-form') as HTMLFormElement;
+    hasStartedLocalVideoTileForm.addEventListener('submit', e => {
+      e.preventDefault();
+      const hasStartedLocalVideoTileValue = this.audioVideo.hasStartedLocalVideoTile();
+      let hasStartedLocalVideoTileBooleanObject = document.getElementById(`has-started-local-video-tile-boolean`);
+      hasStartedLocalVideoTileBooleanObject.innerText = JSON.stringify(hasStartedLocalVideoTileValue, null, '  ');;
+      hasStartedLocalVideoTileForm.reset();
+    });
+
+    const getLocalVideoTileForm = document.getElementById('get-local-video-tile-form') as HTMLFormElement;
+    getLocalVideoTileForm.addEventListener('submit', e => {
+      e.preventDefault();
+      const localVideoTile = this.audioVideo.getLocalVideoTile().id();
+      let videoTileIdObject = document.getElementById(`local-video-tile-id`);
+      videoTileIdObject.innerText = JSON.stringify(localVideoTile, null, '  ');;
+      getLocalVideoTileForm.reset();
+    });
+
+    const getAllVideoTilesForm = document.getElementById('get-all-video-tiles-form') as HTMLFormElement;
+    getAllVideoTilesForm.addEventListener('submit', e => {
+      e.preventDefault();
+      const totalVideoTilesCount = this.audioVideo.getAllVideoTiles().length;
+      let videoTilesCountObject = document.getElementById(`total-video-tiles-count`);
+      videoTilesCountObject.innerText = JSON.stringify(totalVideoTilesCount, null, '  ');;
+      getAllVideoTilesForm.reset();
+    });
+
+    const getAllRemoteVideoTilesForm = document.getElementById('get-all-remote-video-tiles-form') as HTMLFormElement;
+    getAllRemoteVideoTilesForm.addEventListener('submit', e => {
+      e.preventDefault();
+      const totalRemoteVideoTilesCount = this.audioVideo.getAllRemoteVideoTiles().length;
+      let remoteVideoTilesCountObject = document.getElementById(`total-remote-video-tiles-count`);
+      remoteVideoTilesCountObject.innerText = JSON.stringify(totalRemoteVideoTilesCount, null, '  ');;
+      getAllRemoteVideoTilesForm.reset();
+    });
+
+
+    const haveVideoTileForAttendeeIdForm = document.getElementById('have-video-tile-for-attendeeId-form') as HTMLFormElement;
+    haveVideoTileForAttendeeIdForm.addEventListener('submit', e => {
+      e.preventDefault();
+      const attendeeId = this.meetingSession.configuration.credentials.attendeeId;
```

Comment:
I think we should let the user decide what the attendee ID should be, so that we can write dynamic tests

Reply:
Yes you are right. Will make it as an input for dynamic tests